src/routines/write_model.jl

```diff
@@ -102,15 +154,28 @@ function write_model_results(results::OperationModelResults, save_path::String)
     println("Files written to $folder_path folder.")
     return
 end
+""" Exports Simulation Model Results to a path"""
+function write_results(res::OperationsProblemResults, path::String, results::String; kwargs...)
+    if !isdir(path)
+        error("Specified path is not valid. Run write_results to save results.")
+    end
+    folder_path = joinpath(path,results)
+    write_data(res.variables, res.time_stamp, folder_path; kwargs...)
+    write_data(res.optimizer_log, folder_path, "optimizer_log"; kwargs...)
+    time = DataFrames.DataFrame(Range = convert.(Dates.DateTime,res.time_stamp[!,:Range]))
+    write_data(time, folder_path, "time_stamp"; kwargs...)
+    println("Files written to $folder_path folder.")
+    return
+end
 
 """ Exports the OpModel JuMP object in MathOptFormat"""
-function write_op_model(op_model::OperationModel, save_path::String)
-    _write_canonical_model(op_model.canonical, save_path)
+function write_op_problem(op_problem::OperationsProblem, save_path::String)
+    _write_canonical(op_problem.canonical, save_path)
     return
 end
 
 """ Exports the OpModel JuMP object in MathOptFormat"""
-function _write_canonical_model(canonical::CanonicalModel, save_path::String)
+function _write_canonical(canonical::Canonical, save_path::String)
```

Comment:
Why do we need this function? Can we not just copy the contents of this function into `write_op_problem` and remove this `_write_canonical` function?

Reply:
We use _write_canonical also in the simulations. When writing the simulations to disk we call this function among others.